.pre-commit-config.yaml

```diff
@@ -80,29 +80,30 @@ repos:
     hooks:
       - id: helmlint
 
-  - repo: https://github.com/Lucas-C/pre-commit-hooks
-    rev: v1.5.4
-    hooks:
-      - id: insert-license
-        files: '.*\.(py|pyi|yaml|yml|sh|helmignore|dockerignore|gitignore)$'
-        exclude: .devcontainer/tests
-        args:
-          - --license-filepath
-          - license_header.txt
-          - --comment-style
-          - "#"
-          - --skip-license-insertion-comment=This file is maintained by velocitas CLI, do not modify manually.
-      - id: insert-license
-        files: '.*\.(proto)$'
-        args:
-          - --license-filepath
-          - license_header.txt
-          - --comment-style
-          - //
-      - id: insert-license
-        files: "Dockerfile.*"
-        args:
-          - --license-filepath
-          - license_header.txt
-          - --comment-style
-          - "#"
+# Commented out due to insufficient handling of year range in coyright statements
```

Comment:
Please remove it, if it is not necessary. Copy can be recovered from git history.

Reply:
Maybe create a task to fix it or improve in future

Reply:
we already have a bug-ticket for this issue